README.md

```diff
@@ -1,3 +1,50 @@
-# JOD Ohjaaja UI
+# React + TypeScript + Vite
 
-This repository is the UI for JOD Ohjaaja. JOD Ohjaaja is part of the Digital Service Ecosystem for Continuous Learning (JOD) project.
```

Comment:
Maybe using the `JOD Ohjaaja UI` version of the README instead of the template's